lib/phoenix_component.ex

```diff
@@ -1324,6 +1407,7 @@ defmodule Phoenix.Component do
         import Kernel, except: [def: 2, defp: 2]
         import Phoenix.Component
         import Phoenix.Component.Declarative
+        use Phoenix.Template
```

Comment:
I may be wrong but I think this is not necessary at all. `compile_all` should use it before.

Reply:
```suggestion
```